airflow/providers/google/cloud/hooks/cloud_composer.py

```diff
@@ -0,0 +1,402 @@
+        :type region: str
+        :param environment:  The environment to create. This corresponds to the ``environment`` field on the
+            ``request`` instance; if ``request`` is provided, this should not be set.
+        :type environment: google.cloud.orchestration.airflow.service_v1.types.Environment
+        :param retry: Designation of what errors, if any, should be retried.
+        :type retry: google.api_core.retry.Retry
+        :param timeout: The timeout for this request.
+        :type timeout: float
+        :param metadata: Strings which should be sent along with the request as metadata.
+        :type metadata: Sequence[Tuple[str, str]]
+        """
+        client = self.get_environment_client()
+        result = client.create_environment(
+            request={'parent': self.get_parent(project_id, region), 'environment': environment},
+            retry=retry,
+            timeout=timeout,
+            metadata=metadata,
+        )
+        return result
+
+    @GoogleBaseHook.fallback_to_default_project_id
+    def delete_environment(
+        self,
+        project_id: str,
+        region: str,
+        environment_id: str,
+        retry: Optional[Retry] = None,
+        timeout: Optional[float] = None,
+        metadata: Sequence[Tuple[str, str]] = (),
+    ) -> Operation:
+        """
+        Delete an environment.
+
+        :param project_id: Required. The ID of the Google Cloud project that the service belongs to.
+        :type project_id: str
+        :param region: Required. The ID of the Google Cloud region that the service belongs to.
+        :type region: str
+        :param environment_id: Required. The ID of the Google Cloud environment that the service belongs to.
+        :type environment_id: str
+        :param retry: Designation of what errors, if any, should be retried.
+        :type retry: google.api_core.retry.Retry
+        :param timeout: The timeout for this request.
+        :type timeout: float
+        :param metadata: Strings which should be sent along with the request as metadata.
+        :type metadata: Sequence[Tuple[str, str]]
+        """
+        client = self.get_environment_client()
+        name = self.get_environment_name(project_id, region, environment_id)
+        result = client.delete_environment(
+            request={"name": name}, retry=retry, timeout=timeout, metadata=metadata
+        )
+        return result
+
+    @GoogleBaseHook.fallback_to_default_project_id
+    def get_environment(
+        self,
+        project_id: str,
+        region: str,
+        environment_id: str,
+        retry: Optional[Retry] = None,
+        timeout: Optional[float] = None,
+        metadata: Sequence[Tuple[str, str]] = (),
+    ) -> Environment:
+        """
+        Get an existing environment.
+        :param project_id: Required. The ID of the Google Cloud project that the service belongs to.
+        :type project_id: str
```

Comment:
@lwyszomi you should delete `:type` from all docstrings, now it is not necessary

Reply:
👍